packages/quantic/force-app/main/translations/fr.translation-meta.xml

```diff
@@ -700,4 +700,28 @@
     <label>Rechercher à la place {{0}}</label>
     <name>quantic_SearchInsteadFor</name>
   </customLabels>
+  <customLabels>
+    <label>Non pertinent</label>
```

Comment:
@mmitiche @SimonMilord I would consider renaming it "Hors sujet"

Reply:
Hors de propos? 
I didn't find it but I used to say: "Hors propos" might be a quebec-ism